package.json

```diff
@@ -26,6 +26,7 @@
     "vuex": "^3.1.3"
   },
   "devDependencies": {
+    "array.prototype.move": "0.0.4",
```

Comment:
This is probably a prod dependency instead of a dev dependency!

Reply:
Looks like that to me. Updated